pd/src/db.rs

```diff
@@ -14,6 +14,9 @@ pub async fn init_tables(db: &Pool<Postgres>) -> Result<()> {
         query_file!("src/db/notes.sql"),
         query_file!("src/db/nullifiers.sql"),
         query_file!("src/db/validators.sql"),
+        query_file!("src/db/indexes/notes_height_idx.sql"),
+        query_file!("src/db/indexes/notes_position_idx.sql"),
+        query_file!("src/db/indexes/nullifiers_height_idx.sql"),
```

Comment:
what's the motivation to keep the index creation commands separate from the tables they're indexing?

Reply:
as far as I can tell, this is a limitation in the existing `query_file!` pattern - I would like to combine them, but `query_file!` is apparently generating prepared statements which forbid multiple commands in a single file. @plaidfinch do you happen to know an alternative way to do this that might let us consolidate the create table and index creation files?

Reply:
No need to fix it, I was just curious!

Reply:
@aubrey-dyer The alternative way would be to use `sqlx`'s explicit migration infrastructure: https://docs.rs/sqlx/0.5.9/sqlx/macro.migrate.html. Perhaps we can look into that?